packages/aws-cdk/lib/api/bootstrap/bootstrap-template.yaml

```diff
@@ -296,6 +299,9 @@ Resources:
             Version: '2012-10-17'
           PolicyName:
             Fn::Sub: cdk-${Qualifier}-lookup-role-default-policy-${AWS::AccountId}-${AWS::Region}
+      Tags:
+        - Key: aws-cdk:bootstrap-role
+          Value: image-publishing
```

Comment:
This one looks to be attached to the wrong role?

Reply:
Whoops. Bad merge conflict resolution, I think.